extension/src/ctp/ctp.js

```diff
@@ -66,9 +72,19 @@ function setUpClient () {
         this.buildRequestOptions(uri.byId(id).build(), 'POST', body)
       )
     },
+
     fetch (uri) {
       return ctpClient.execute(this.buildRequestOptions(uri.build()))
     },
+
+    fetchBatches (uri, cb, opts = { accumulate: false }) {
+      return this.process(
+        this.buildRequestOptions(uri.build()),
+        data => Promise.resolve(cb(data.body.results)),
```

Comment:
`Promise.resolve()` is a native function. No need for bluebird

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/5/commits/f37d84606eaf6a2ee4fdff3ef77b3ff8767f5e5c